bgflow/nn/flow/crd_transform/ic.py

```diff
@@ -745,9 +702,9 @@ def _forward(self, x, *args, **kwargs):
         # aggregate volume change
         dlogp = dlogp_rel + dlogp_ref
 
-        return bonds, angles, torsions, x0, R, dlogp
+        return bonds, angles, torsions, x0, alpha, beta, gamma, dlogp
```

Comment:
This is a breaking change. Can we at least ensure to use the same number of output arguments. That would mean returning a tensor `euler_angles:=(alpha, beta, gamma)` instead of the individual angles; 
also please document the range of these angles

Reply:
Alright - I will unify it then and implement the options to return both by choice: `R` as a basis frame, `euler` as 3-vector of Euler angles. And yeah - will add the range to the doc.